recipes-bsp/acpi-tables/acpi-tables.bb

```diff
@@ -14,6 +14,14 @@ inherit deploy
 
 ACPI_TABLES ?= ""
 ACPI_TABLES[doc] = "List of ACPI tables to include with the initrd"
+ACPI_FEATURES_edison ?= "uart_2w spi i2c"
+IASLFLAGS = " \
+    ${@bb.utils.contains('ACPI_FEATURES', 'uart_2w', '-D MUX_UART_2WIRE', '', d)} \
```

Comment:
If it works in a form `-DMUX_UART_2WIRE`  (no space) and so on for the rest, I would prefer it over spaceful -version.

Reply:
Tested and done.